newt/install/install.go

```diff
@@ -647,17 +647,29 @@ func (inst *Installer) calcVersionMap(candidates []*repo.Repo) (
 	}
 
 	// If project.yml specified any specific git commits, ensure we get them.
```

Comment:
This comment could be updated to indicate that we are checking all repo dependencies (not just those from `project.yml`).